schedule/qam/common/qam-yast_self_update_libyui-rest.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: qam-yast_self_update_libyui-rest
+description: self_update via INSTALLER repo & efi test on UEFI
```

Comment:
could we please a less cryptic description, something like `installation using self_update as boot parameter. We don't need to mention in the yaml schedule description, the ARCH or UEFI. we usually forget to remove it when we separate in different files.

Reply:
good hint, I've changed the description; thanks!